cmd/call.go

```diff
@@ -0,0 +1,29 @@
+/*
+ * SPDX-License-Identifier: AGPL-3.0-only
+ * Copyright (c) 2022-2023, daeuniverse Organization <dae@v2raya.org>
+ */
+
+ package cmd
+
+ import (
+	 "fmt"
+	 "os"
+ 
+	 "github.com/spf13/cobra"
+ )
+ 
+ var (
+	 honkCmd = &cobra.Command{
+		 Use:   "honk",
+		 Short: "Let dae call for you.",
+		 Run: func(cmd *cobra.Command, args []string) {
+				 fmt.Println("Honk! Honk! Honk! This is dae!")
+				 os.Exit(1)
```

Comment:
1 indicates errors. We should use os.Exit(0) here.

BTW, do you have tested it? Please paste the result. Thank you.

Reply:
With exit(1), it works like:
![image](https://github.com/daeuniverse/dae/assets/17946453/c72ae686-6e05-476b-a36a-3f91cd694494)
And it was changed to exit(0) in this fix commit.